daffodil-cli/README_How_to_Debug.txt

```diff
@@ -1,15 +0,0 @@
-Because the CLI tests spawn a sub-shell and exec the daffodil program
```

Comment:
This content was consolidated into daffodil-cli/README.md since Apache Rat doesn't realize this filename as a readme and complains about missing apache license. However, the content probably doesn't actually belong in the Daffodi CLI Readme, since that is more about how to run the CLI released in zip/tar/rpm. This probably really belongs in the root README cli since that talks about how to run cli tests and it more related to source that a release.